docs/get-involved/delegation-policy.md

```diff
@@ -1,61 +1,148 @@
 # Delegation Policy
 
-Oasis Protocol Foundation delegates ROSE tokens to node operators based on their
-participation in the Oasis Mainnet, Testnet and ParaTimes on the network,
-node reliability and performance, community engagement and overall support of
-the network.
-
-This page describes the Oasis Protocol Foundation's Delegation Policy.
+The Oasis Protocol Foundation (OPF) delegates ROSE tokens to node operators who
+run Oasis Consensus and ParaTime nodes on Mainnet and/or Testnet. Delegations
+are based on node reliability, performance, community engagement, and overall
+contributions to the network’s growth and stability.
 
 ## Requirements for Receiving Delegations
 
+To be eligible for a delegation, you have to join the network, follow the Code
+of Conduct, and meet performance standards.
+
+For guidance on how to increase your delegation over time, please refer to the
+instructions at the bottom of this page.
+
+### Join the Network
+
+1. **Join the Community Channels**
+   - Join the `#node-operators` channel on the [Oasis Discord server]
+   - Or join the Telegram group: [@oasisnodeoperators]
+
+2. **Create Your Entity Keys**
+   - Generate your **Testnet** and **Mainnet** [entity keys]
+
+3. **Set Up Your Nodes**
+   - [Configure and start] your Testnet Consensus and ParaTime nodes
+   - [Configure and start] your Mainnet Consensus and ParaTime nodes
+
+4. **Submit Your Metadata**
+   - [Provide metadata] for both **Mainnet** and **Testnet**
+
+5. **Reach Out to the Node Operator Relationship Manager**
+   - Message `@am3lody` on Discord or Telegram
+   - Be prepared to answer any follow-up questions
+
+6. **Stay Updated**
+   - Follow network announcements:
+      - 📢 [Oasis Discord server]: `#mainnet-announcements` and
+      `#testnet-announcements`
+      - 📢 [Oasis Mainnet Announcements]
+      - 📢 [Oasis Testnet Announcements]
+      - 📢 [Oasis Node Operators Group]
```

Comment:
This is a bit puzzling. I'd have two bullets, on for Discord and another for TG.

```
- Follow one of our social media channels for network announcements:
  - Discord: ...
  - Telegram: ...
```